tools/azure-sdk-tools/gh_tools/dev_requirements.txt

```diff
@@ -0,0 +1 @@
+azure-identity
```

Comment:
If we create a dev_reqs for this directory, we should include the other necessary reqs as well